src/dcmstack/dcmstack_cli.py

```diff
@@ -250,10 +250,10 @@ def main(argv=sys.argv):
     #Handle each source directory individually
     for src_dir in args.src_dirs:
         if not os.path.isdir(src_dir):
-            print >> sys.stderr, '%s is not a directory, skipping' % src_dir
+            print('%s is not a directory, skipping' % src_dir, file=sys.stderr)
```

Comment:
You need to do ``from __future__ import print_function`` in order for this to work on Python 2 as well.